docs/2.9. Usage - Fully Automated Testing of NMOS Controllers.md

```diff
@@ -0,0 +1,174 @@
+# Testing NMOS Controllers
+
+## Fully Automated NMOS Controller Tests
```

Comment:
Does this work well with the non-interactive way of running the testing tool, passing the suite name, IP addresses/ports and API versions, etc. on the command line? That's great if so!

Reply:
Admittedly I'm unsure whether this has been explicitly tested - in theory it should work, but will need verification

Reply:
I have tested this, both test suites work when run from the command line.